sendfile-Attachments/workflows/src/main/kotlin/com/sendfile/flows/downloadAttachment.kt

```diff
@@ -0,0 +1,61 @@
+package com.sendfile.flows
+
+import co.paralleluniverse.fibers.Suspendable
+import com.sendfile.states.InvoiceState
+import net.corda.core.crypto.SecureHash
+import net.corda.core.flows.*
+import net.corda.core.identity.Party
+import net.corda.core.node.services.vault.QueryCriteria
+import net.corda.core.utilities.ProgressTracker
+import java.io.File
+import java.io.InputStream
+
+// *********
+// * Flows *
+// *********
+@InitiatingFlow
+@StartableByRPC
+class downloadAttachment(
```

Comment:
Probably best to stick to the naming convention for classes (so uppercase the first word, like `DownloadAttachment`). Similarly for other classes in this app.